### DISCUSSION TITLE
Cfast is very Slow for Preview

### DISCUSSION THREAD

**Discussion by marcoelec81** [firemodels/cfast#2108] [General]
Hi Guys, I installed the version CFAST 7.7.4, Smokeview 6.7.21, but at the moment for Preview the Compartement, Wall and Vents, the software delay between 45 to 63 seconds for a simple geometry, my compurer is a Lenovo E15 wit RYZEN 7 an AMD RADEON Chipset, Is there a configuration that can solve the problem?

**Comment by mcgratta** [comment]
Is there another computer with which you can test this same case?

**Reply by marcoelec81** [reply]
Yes, of course, but they have similar hardware characteristics, it has AMD processors, will there be any problems with these processors? I see that the compiler is configured only for Intel, there is also a message at the beginning that says the following: "A print device has not It has been found that it can be used to measure visual elements, this can negatively affect the quality of the representation.

**Reply by mcgratta** [reply]
Could you tell me exactly what the message is and what is producing it. I do not understand what you mean when you say a "message at the beginning..."

**Reply by marcoelec81** [reply]
Hi Bro, I have seen that the SmokeView execution mode has the following message in the CMD: "research mode on, using global bounds", but in a tutorial on Youtube you can see another command line in which you can see the execution of GLUT by OPENGL .
The message that it showed you previously came out in Spanish and I tried to translate it as best as possible into English, I still uninstalled and reinstalled and it didn't appear again.
![cap_1](https://github.com/firemodels/cfast/assets/36569993/8a321eee-2127-4536-a0c7-fc2b9acbdb1a)

There is a considerable difference in the command line executed by the tutorial on YouTube, could this be the origin of the slowness when executing SmokeView?

![image](https://github.com/firemodels/cfast/assets/36569993/0cdfcdd7-1f02-4a29-94bf-e1c45c791b11)

![image](https://github.com/firemodels/cfast/assets/36569993/46636405-2533-4ed1-9052-9e0f5f314611)

**Reply by drjfloyd** [reply]
DIfferent versions of Smokeview. At some point (I forget exactly when) we switched from having Smokeview pick bounds for displaying data at 1 % and 99 % of the values of data loaded to just having Smokeview use the max min of all the data read in -- too many users not realizing that what was being done and wondering why max values were too low. That had been called "research mode" in older version of Smokeview. We just made that the default but didn't change the messaging scrolling to the Smokeview log on the screen.

**Reply by marcoelec81** [reply]
But, is there a configuration so that Smoke View displays the graphic results faster?

**Reply by gforney** [reply]
Upload the case that is slow for you and ill take a look

**Reply by marcoelec81** [reply]
[Burn_Building_V2_zone.csv](https://github.com/firemodels/cfast/files/15370258/Burn_Building_V2_zone.csv)
[Burn_Building_V2_walls.csv](https://github.com/firemodels/cfast/files/15370259/Burn_Building_V2_walls.csv)
[Burn_Building_V2_vents.csv](https://github.com/firemodels/cfast/files/15370260/Burn_Building_V2_vents.csv)
[Burn_Building_V2_masses.csv](https://github.com/firemodels/cfast/files/15370261/Burn_Building_V2_masses.csv)
[Burn_Building_V2_devices.csv](https://github.com/firemodels/cfast/files/15370262/Burn_Building_V2_devices.csv)
[Burn_Building_V2_compartments.csv](https://github.com/firemodels/cfast/files/15370263/Burn_Building_V2_compartments.csv)

**Reply by marcoelec81** [reply]
I'm sorry, I couldn't find a way to upload the .ini file, I tried to send it compressed but it won't let me send it

**Reply by drjfloyd** [reply]
If you add .txt to the filename and you should be able to attach it  

casename.ini -> casename.ini.txt

**Reply by gforney** [reply]
All i need is the input file, ie the .in file

**Reply by marcoelec81** [reply]
Ok, I'll do it right away, thanks
[Burn_Building_V2.log](https://github.com/firemodels/cfast/files/15396535/Burn_Building_V2.log)

**Comment by marcoelec81** [comment]
[Burn_Building_V2.log](https://github.com/firemodels/cfast/files/15396563/Burn_Building_V2.log)

Is this!, ;)

**Comment by gforney** [comment]
I need the cfast input file not the log file.
